vmdb/app/models/miq_report/search.rb

```diff
@@ -14,7 +14,7 @@ def get_limit_offset(page, per_page)
   end
 
   def get_sqltable(assoc)
-    r = self.db_class.reflect_on_association(assoc.to_sym)
+    r = db_class.reflection_with_virtual(assoc.to_sym)
```

Comment:
So, @Fryguy I think this should be ok :confused:  ...  We're only calling get_sqltable when we have sortby columns like "some_association.some_column" where the changed behavior is that some_association can be a virtual reflection.  I guess this should just work?